README.md

```diff
@@ -242,18 +242,85 @@ Also missing are the following.
 
 ## Installing the reference implementation
 
+Currently, there are two ways of installing the reference implementation.
+You may install it on an existing Raspbian system or create a new image.
+The two alternatives are presented below.
+
+### Install on an existing system
+
 You'll need to install the required libraries and project.
 Here is how you can do it under the Ubuntu GNU/Linux distribution.
 
 ```bash
 sudo apt-get install git libbluetooth-dev libglib2.0-dev python3-dev python3-setuptools shellcheck sqlite3 virtualenv dh-virtualenv debhelper supervisor
-git clone https://github.com/dspinellis/epidose
+git clone https://github.com/eellak/epidose
 cd epidose
 virtualenv venv -p /usr/bin/python3
 . venv/bin/activate
 pip3 install -e .
 ```
 
+### Create a new image
+
+You'll need to download a vanilla Raspbian distribution (Lite version is suggested for this purpose)
+and burn it on a microSD card.
+Then perform the following steps on your microSD card while it is mounted on your personal computer.
+
+* [Enable the SSH daemon](https://howtoraspberrypi.com/how-to-raspberry-pi-headless-setup/)
+* Add your WiFi network's SSID and PSK in the [wpa_supplicant.conf file](https://howtoraspberrypi.com/how-to-raspberry-pi-headless-setup/)
+* Assign a static IP address to your Raspberry-Pi system by adding the following lines in the rootfs/etc/dhcpcd.conf file (found in you microSD card) according to your router settings:
+```bash
+interface wlan0
+static ip_address=192.168.0.[2-254]/24
+static routers=192.168.0.1
+static domain_name_servers=192.168.0.1
+```
+* Copy the wpa_supplicant.conf from the rootfs/etc/wpa_supplicant/wpa_supplicant.conf to the boot/ directory of the new image to unblock to WiFi:
```

Comment:
I think it's best to do it in the opposite way.  Modify only the files in `/boot`, which is visible to Windows and macOS computers as well, and then this gets automatically propagated during boot.  Can you please verify and adjust?

Reply:
Ok, I will try to burn a new image on a Windows system and add the relevant instructions.
For macOS, I will discuss it with Konstantinos Kravvaritis.

About the setting a static IP, AFAIK you can add it in the /rootfs/etc/dhcpcd.conf.
Is there a way to do that on the /boot partition?

Reply:
Why do you need a static IP address?  Why not let it obtain an address over DHCP and simplify the process?